tsfeatures/tsfeatures.py

```diff
@@ -835,109 +855,96 @@ def unitroot_pp(x: np.array, freq: int = 1) -> Dict[str, float]:
     except:
         test_pp = np.nan
 
-    return {'unitroot_pp': test_pp}
-
-def statistics(x: np.array, freq: int = 1) -> Dict[str, float]:
```

Comment:
This function seems to be missing

Reply:
added the statistics function

Reply:
You forgot the `#| export` for this function

Reply:
added the export directive